docs/user/checks.rst

```diff
@@ -35,6 +35,16 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+.. _data_collected_check:
+
+Data Collected
+--------------
+
+This check ensures that the :doc:`openwisp-monitoring agent
+</openwrt-monitoring-agent/index>` is running and sending passive metrics
```

Comment:
"sending passive metrics" doesn't make sense IMHO: the agent sends metrics, the server receives it, so the server passively receives them, but the agent actively sends them, so saying the agent sends passive metrics seems contradictory to me.